### ISSUE TITLE
Auto height & navbar combo causes unneeded scrollbars

### ISSUE THREAD

**Issue by ockhamdesign** [FrDH/mmenu-js#1048]
The combination of autoheight and navbars in options creates an unneeded scrollbar.  Basically, I just want to add the closing X to the navbar and use autoheight without a scrollbar.

```
<nav id="my-menu">
    <ul>
        <li><a href="/">Home</a></li>
        <li><a href="/about/">About us</a>
            <ul>
                <li><a href="/about/history/">History</a></li>
                <li><a href="/about/team/">The team</a></li>
                <li><a href="/about/address/">Our address</a></li>
            </ul>
        </li>
        <li><a href="/contact/">Contact</a></li>
    </ul>
</nav>
```

JS - 

        document.addEventListener(
            "DOMContentLoaded", () => {
                new Mmenu( "#my-menu", {
                    // options
                    "extensions": ["position-back","position-top"],
                    "navbar": {"title":"Main Menu"},
                    "autoHeight": true,
                    "navbars": [{
                        "position": "top",
                        "content": ["prev","title","close" ]
                                }]
                }, {
                    // configuration
                    offCanvas: {
                        // page: {
                        //     nodetype: "#my-page"
                        // }
                    },
                });
            }
        );

**Comment by FrDH** [comment]
Thank you for reporting the issue.
Will be fixed in the next release.

**Comment by FrDH** [comment]
Should be fixed in 8.3
